resources/Hooks/useConditionalAutofocus.js

```diff
@@ -0,0 +1,10 @@
+import { useRef } from 'react'
+
+const useConditionalAutofocus = (initialValue, useAutofocus, timeout = 450) => {
```

Comment:
like the idea, but can we rename it to something a little but less of a mouthful? even just `useAutofocus` should be good I think

Reply:
also how come this hook takes 3 params but always gets called w. 2?

Reply:
and also if we never pass in an initial value besides null maybe we should just remove those params and have the only param be the timeout